Update View Astro Clusters to Match New UI Format (#1799)
astronomer/docs: astro/view-clusters.md

| Created            | The date and time the cluster was created                                                                                   |

The **Worker Types** tab displays the following information:

| Value              | Description                                                                                                                 |
| ------------------ | --------------------------------------------------------------------------------------------------------------------------- |
| Node instance type | The instance type used for the cluster's worker nodes                                                                       |
| ID                 | The ID of the node instance type used for the cluster's worker nodes                                                        |
| Max node count     | The maximum number of worker nodes supported across all Deployments in the cluster                                          |

The **Workspace Authorization** tab lets you define what Workspaces can create Deployments on the cluster.

> The **Tags** tab displays the following information about the custom tags created by Astronomer support:

Reviewer: ```suggestion
The **Tags** tab displays custom tags for the cluster created by Astronomer support.
```
I think this makes more sense to keep this brief and remove the table. It's not "information about" the tags, its the tags themselves.
